builds/codex-v103-aeturnum-monorepo/packages/core/src/codex_core/helix.py

```
@@ -0,0 +1,23 @@
+import hashlib, json
+class TetraHelix:
+    def __init__(self):
+        self.strands = {k: [] for k in ("logic","governance","learning","ethics")}
+    @staticmethod
+    def _h(b: bytes)->str: return hashlib.sha256(b).hexdigest()
+    def add_event(self, strand:str, payload:dict):
+        if strand not in self.strands: raise ValueError("unknown strand")
```

Ruff (0.14.3)

Multiple statements on one line (colon)

(E701)